src/example-types/metricGroups/pivot.js

```diff
@@ -124,16 +191,73 @@ let getSortField = ({ columnValues = [], valueProp, valueIndex } = {}) =>
     valueProp,
   ])
 
+let mapDrilldownPages = node => {
+  let pagination = node.pagination || { columns: {}, rows: {} }
+  let isDrilldown = _.flow(
+    _.flatMapDeep(type =>
+      _.map(prop => _.get([type, prop], pagination), ['drilldown', 'skip'])
+    ),
+    _.some(_.negate(_.isEmpty))
+  )(['columns', 'rows'])
+
+  let rowDrillMode = !(
+    _.isEmpty(pagination.rows.drilldown) && _.isEmpty(pagination.rows.skip)
+  )
+  let expanded = rowDrillMode
+    ? _.get('columns.expanded', pagination)
+    : _.get('rows.expanded', pagination)
```

Comment:
this seems confusing since nodes could have both column and row expansions

Reply:
When you are drilling down a row you want to perform it against all expanded columns and vice-versa. So when we expanding another row we don't care about rows previously expanded only columns